src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -154,17 +156,20 @@ def _assess_clusters(self, all_clusters):
         for cluster in all_clusters:
             if cluster.cluster_source == ClusterSource.JOB:
                 continue
-            if not cluster.creator_user_name:
+            creator = cluster.creator_user_name
+            if not creator:
                 logger.warning(
                     f"Cluster {cluster.cluster_id} have Unknown creator, it means that the original creator "
                     f"has been deleted and should be re-created"
```

Comment:
should the User be recreated or the cluster?

Reply:
The cluster should either have its owner changed (there is an API for this) or be recreated.

Interestingly there's a mix of terminology here: the property here is `creator_user_name` but the REST API for changing it is `…/clusters/change-owner`, and such requests have a `owner_username` property.